### PR TITLE
Add an example automation to myuplink page

### PR DESCRIPTION
**Pull request by astrandb**
## Proposed change

Add an example automation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Updated the "Use cases" section for the myUplink integration.
	- Added a new "Example" section with a YAML automation example for low hot water notifications.
	- Clarified the integration's polling behavior, specifying a 60-second API polling interval.
	- Retained the known limitations section regarding entity mapping and static entity names.

### CODE REVIEW

**Review by andrewsayre — APPROVED** [review]

File: source/_integrations/myuplink.markdown [hunk 1]
```diff
@@ -77,6 +77,27 @@ Common use cases include:
...
+      entity_id:
+        - sensor.your_pump_hot_water_charging_bt6
+      below: 42
+  actions:
+    - action: notify.mobile_app_your_device
+      metadata: {}
+      data:
+        message: Hot water reserve is getting low.
+        title: Water heater
+  mode: single
```
Comment (frenck) [inline source/_integrations/myuplink.markdown:98]: This mode is the default, and should thus not be added to the examples.

File: source/_integrations/myuplink.markdown [hunk 2]
```diff
@@ -77,6 +77,27 @@ Common use cases include:
 - **Automation**: Adjust the temperature curve offset during holiday mode
 - **Analytics**: View long-term statistics and graphs for the relevant sensors
 
+## Example
+
+Automation that will send a notification to a smartphone when the hot water reserve is getting low. Note that actual entity name varies between models of heat pumps. You will have to adapt the yaml code to your own installation.
+
+```yaml
+automation:
+  alias: Notify on low hot water reserve
```
Comment (frenck) [inline source/_integrations/myuplink.markdown:86]: Strings should be enquoted, see also our YAML style guide:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

This comments applies to multiple items in this example.

Comment (frenck) [inline source/_integrations/myuplink.markdown:86]: Automation is a list, this automation is thus expected to be an item in that list of automations.

File: source/_integrations/myuplink.markdown [hunk 3]
```diff
@@ -77,6 +77,27 @@ Common use cases include:
...
+  triggers:
+    - trigger: numeric_state
+      entity_id:
+        - sensor.your_pump_hot_water_charging_bt6
+      below: 42
+  actions:
+    - action: notify.mobile_app_your_device
+      metadata: {}
```
Comment (frenck) [inline source/_integrations/myuplink.markdown:94]: Empty data, can be removed.

**Review by frenck — COMMENTED** [review]
Hey @astrandb some things sneaked in, that shouldn't have. Could you take a look?

/CC @andrewsayre

**Pull request merged**